nautobot_chatops/urls.py

```diff
@@ -25,19 +23,6 @@
     CommandTokenView,
 )
 
-if get_app_settings_or_config("nautobot_chatops", "enable_grafana"):
-    try:
-        from nautobot_chatops.integrations.grafana.urls import urlpatterns as grafana_urlpatterns
-    # pylint: disable-next=broad-except
-    except Exception:
-        grafana_urlpatterns = []
-        logger = logging.getLogger(__name__)
-        logger.warning("Grafana ChatOps integration is not available.", exc_info=True)
```

Comment:
Wondering - were there cases where importing the grafana URLs might fail here (missing optional dependency, etc.) that we need to handle still?

Reply:
Probably. Our dockerfile installs with `--extras all`. I'll test without that.

Reply:
Looks like I need to add more import guardrails 😞 

If I just put this exception handling back in we get a broken nav menu.

![image](https://github.com/user-attachments/assets/314e76b6-ba08-4c5c-9f2a-8f3f937f7d1c)

Reply:
This is fixed now. Discussed the implementation in DSU and decided not to spend too much time on it since ideally nobody will ever encounter this.